### PR TITLE
cleanup component install to use new catalog feature

### PR DESCRIPTION
**Pull request by zehicle**
Closes: #14062
The process of the install has not changed.  This fix is to cleanup component install instructions to use new DRP catalog feature so that we don't have to hard code a version reference.

### CODE REVIEW

**Comment by sygibson** [comment]
+1

File: content/en/docs/setup/on-premises-metal/krib.md [hunk 1]
```diff
@@ -38,7 +38,12 @@ Following the [Digital Rebar installation](https://provision.readthedocs.io/en/t
 
 ### (2/5) Install KRIB Content and Certificate Plugin
 
-Upload the KRIB Content bundle (or build from [source](https://github.com/digitalrebar/provision-content/tree/master/krib)) and the Cert Plugin for your DRP platform (e.g.: [amd64 Linux v2.4.0](https://s3-us-west-2.amazonaws.com/rebar-catalog/certs/v2.4.0-0-02301d35f9f664d6c81d904c92a9c81d3fd41d2c/amd64/linux/certs)). Both are freely available via the [RackN UX](https://portal.rackn.io).
+Upload the KRIB Content bundle (or build from [source](https://github.com/digitalrebar/provision-content/tree/master/krib)) and the Cert Plugin for your DRP platform. Both are freely available via the [RackN UX](https://portal.rackn.io) or using the upload from catalog feature of the DRPCLI (shown below).
+
+  ::
```
Comment (tengqm) [inline content/en/docs/setup/on-premises-metal/krib.md]: Line 43 looks like ReST grammar rather than markdown ...

**Pull request merged**